src/main/kotlin/xyz/bluspring/kilt/loader/remap/KiltRemapper.kt

```diff
@@ -148,444 +177,475 @@ object KiltRemapper {
 
         val exceptions = mutableListOf<Exception>()
 
-        val modRemapQueue = ArrayList(modLoadingQueue)
-
         // Sort according to what dependencies are required and should be loaded first.
-        // If a mod fails to remap because a dependency isn't listed, welp,
-        // that's their problem now i guess.
-        modRemapQueue.sortWith { a, b ->
-            if (a.dependencies.any { it.modId == b.modId })
-                1
-            else if (b.dependencies.any { it.modId == a.modId })
-                -1
-            else 0
-        }
+        // If a mod fails to remap because a dependency isn't listed, well,
+        // that's their problem now, I guess.
```

Comment:
I don't think this comment is needed anymore, seeing as the mods aren't being sorted anymore.